test/hotspot/jtreg/runtime/ClassUnload/UnloadTest.java

```diff
@@ -48,6 +48,8 @@
  *    and verifies the class is unloaded.
  */
 public class UnloadTest {
+    // JDK-8297740: using a global static field to keep the object live in -Xcomp mode.
```

Comment:
No need to reference the bug in the comment - we don't generally do that. You could add it to `@bug` list.